units/initrd-battery-check.service

```diff
@@ -0,0 +1,20 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Check battery level during early boot
+DefaultDependencies=no
+After=systemd-udevd.service
+Before=initrd-switch-root.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/lib/systemd/systemd-battery-check
```

Comment:
This unit will need to be templated, since the `/usr/lib/systemd/` path might differ on different distros, i.e. rename it to `units/initrd-battery-check.service.in`, change this line to:

```suggestion
ExecStart={{ROOTLIBEXECDIR}}/systemd-battery-check
```

and in `units/meson.build` it should be added to the "in_units" list instead of "units" so it's correctly renamed and templated during build.

Reply:
> (You can also locally check if everything works correctly by doing the usual `meson build && ninja -C build` loop and then checking if the final unit exists as `build/units/initrd-battery-check.service` and has `{{ROOTLIBEXECDIR}}` replaced with an actual path).

Alright sir. Thank you so much.

Reply:
> (You can also locally check if everything works correctly by doing the usual `meson build && ninja -C build` loop and then checking if the final unit exists as `build/units/initrd-battery-check.service` and has `{{ROOTLIBEXECDIR}}` replaced with an actual path).

I just finished building on my ubuntu.

I ran ```cat build/units/initrd-battery-check.service```

the ```ExecStart={{ROOTLIBEXECDIR}}/systemd-battery-check``` got replaced.

I had ```ExecStart=/usr/lib/systemd/systemd-battery-check```